<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPainterPerf</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Source.qrc">
    <normaloff>:/images/image/image-for-test.png</normaloff>:/images/image/image-for-test.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabMain">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabInfo">
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonReset">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelRounds">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Test rounds:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinRounds">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="comboTests">
          <item>
           <property name="text">
            <string>(0)All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(1)QPainter - Points</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(2)QPainter - Lines</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(3)QPainter - CONVEXPOLYGON</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(4)QPainter - ELLIPSE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(5)QPainter - IMAGE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(6)QPainter - PATH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(7)QPainter - PICTURE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(8)QPainter - PIE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(9)QPainter - PIXMAP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(10)QPainter - ARC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(11)QPainter - CHORD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(12)QPainter - POLYGON</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(13)QPainter - POLYLINE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(14)QPainter - RECT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(15)QPainter - ROUNDEDRECT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(16)QPainter - TEXT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(17)QPainter - STATICTEXT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(18)QPainter - TILEDPIXMAP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(19)QPainter - DRAWGLYPHRUN</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(20)QPainter - FILLPATH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(21)QPainter - ERASE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(22)QPainter - FILLRECT</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="buttonRun">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textConsole"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDrawing">
       <attribute name="title">
        <string>Drawing</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelQPainter">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>QPainter</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="TestWidget" name="testDrawWidget" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TestWidget</class>
   <extends>QWidget</extends>
   <header>testwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Source.qrc"/>
 </resources>
 <connections/>
</ui>
